docs/data-types.md

```diff
@@ -0,0 +1,69 @@
+# Data Types
+
+Postman Flows in its heart is a dataflow language, which means that there flows
+understands all kinds of values and associates a type to data. It should also be
+noted that Flows in a hybrid typed language, which means that it perform static 
+type checking to provide features like auto-complete and early warning, and it 
+also performs some dynamic type checking during execution. Our intention, in the long run, is to move Flows towards being
+strictly typed language over a period of time. 
+
+We are assuming most flow programmers would either be coming from a 
+javascript/typescript background, or it might be their first time programming in 
+an API-First language; therefore we have tried to keep the vocabulary of the 
+types close to javascript/typescript.
+
+## Primitive Values
+| type | What it accepts | Example|
+|---------|-----------------|--------|
+|bool     | **true** or **false** | `true`|
+|string   | utf-8 encoded characters |`foo 😎 bar` |
+|number   | double precision 64-bit values or 64bit unsigned integers| `3.14` <br> `4294967295`|
```

Comment:
Should there be a floating point int example as well?

Reply:
`3.14` is floating point and `4294967295` is integer